newsroom/templates/download_agenda.txt

```diff
@@ -6,7 +6,7 @@
 {% endif %}
 {%- if item.definition_long %}Definition: {{ item.definition_long }}
 {% endif -%}
-On: {{ item.dates.start | datetime_long }}
+On: {{ (item.dates.end or item.dates.start) | datetime_long }}
```

Comment:
shouldn't this be rather the other way around?

Reply:
it's not showing that because of the end time but due to the timezone, it starts at 18:30 and the timezone is +5:30, so it makes that start being on 18th at 00:00 local time (which is what we display in the UI)

Reply:
I see, I will update it